Source/WebCore/css/parser/CSSParserContext.cpp

```diff
@@ -154,7 +155,8 @@ void add(Hasher& hasher, const CSSParserContext& context)
         | context.targetTextPseudoElementEnabled            << 26
         | context.viewTransitionTypesEnabled                << 27
         | context.cssProgressFunctionEnabled                << 28
-        | (uint32_t)context.mode                            << 29; // This is multiple bits, so keep it last.
+        | context.cssHasSlottedEnabled                      << 29
+        | (uint32_t)context.mode                            << 30; // This is multiple bits, so keep it last.
```

Comment:
CSSParserMode has 5 values and requires 3 bits so this won't work. We probably need to get rid of one of the existing flags in order to do this.

Reply:
Maybe we should get rid of customStateSet. It has been stable & shipping for a while.